cmd/ocfclient/main.go

```diff
@@ -35,59 +42,27 @@ type Options struct {
-	fmt.Println("    --identityCert=<Identity Certificate> i.e. end_cert.crt")
-	fmt.Println("    --identityKey=<Identity Certificate> i.e. end_cert.key")
-	fmt.Println("    --identityIntermediateCA=<Identity Intermediate CA Certificate> i.e. subca_cert.crt")
-	fmt.Println("    --identityIntermediateCAKey=<Identity Intermediate CA Private Key> i.e. subca_cert.key")
-	fmt.Println("    --identityTrustCA=<Identity Trusted CA Certificate> i.e. rootca_cert.crt")
-	fmt.Println("    --identityTrustCA=<Identity Trusted CA Private Key> i.e. rootca_cert.key")
+	fmt.Println("    --certIdentity=<Device UUID>             i.e. 00000000-0000-0000-0000-000000000001")
+	fmt.Println("    --mfgCert=<Manufacturer Certificate>     i.e. mfg_cert.crt")
+	fmt.Println("    --mfgKey=<Manufacturer Private Key>      i.e. mfg_cert.key")
+	fmt.Println("    --mfgTrustCA=<Manufacturer Trusted CA Certificate>         i.e. mfg_rootca.crt")
+	fmt.Println("    --mfgTrustCAKey=<Manufacturer Trusted CA Private Key>      i.e. mfg_rootca.key")
+	fmt.Println("    --identityCert=<Identity Certificate>      i.e. end_cert.crt")
+	fmt.Println("    --identityKey=<Identity Certificate>       i.e. end_cert.key")
+	fmt.Println("    --identityIntermediateCA=<Identity Intermediate CA Certificate>     i.e. subca_cert.crt")
+	fmt.Println("    --identityIntermediateCAKey=<Identity Intermediate CA Private Key>  i.e. subca_cert.key")
+	fmt.Println("    --identityTrustCA=<Identity Trusted CA Certificate>     i.e. rootca_cert.crt")
+	fmt.Println("    --identityTrustCA=<Identity Trusted CA Private Key>     i.e. rootca_cert.key")
 	fmt.Println()
 
-	// Mfg Certificates
-	if opts.MfgCert != "" {
-		mfgCert, err := ioutil.ReadFile(opts.MfgCert)
-		if err != nil {
-			fmt.Println("Reading MfgCert was failed : " + err.Error())
-		} else {
-			fmt.Println("Reading MfgCert from file (" + opts.MfgCert + ") was successful")
-			MfgCert = mfgCert
-		}
-	} else {
-		filename := "mfg_cert.crt"
-		if !fileExists(filename) {
-			err := ioutil.WriteFile(filename, MfgCert, 0644)
-			if err != nil {
-				fmt.Println("Writing MfgCert was failed : " + err.Error())
-			} else {
-				fmt.Println("Writing MfgCert to "+filename+" for sample use ...")
-			}
-		}
-	}
-
-	if opts.MfgKey != "" {
-		mfgKey, err := ioutil.ReadFile(opts.MfgKey)
-		if err != nil {
-			fmt.Println("Reading MfgKey was failed : " + err.Error())
-		} else {
-			fmt.Println("Reading MfgKey from file (" + opts.MfgKey+") was successful")
-			MfgKey = mfgKey
-		}
+	// Certificate Identity
+	if opts.CertIdentity != "" {
+		CertIdentity = opts.CertIdentity
 	} else {
-		filename := "mfg_cert.key"
-		if !fileExists(filename) {
-			err := ioutil.WriteFile(filename, MfgKey, 0644)
-			if err != nil {
-				fmt.Println("Writing MfgKey was failed : " + err.Error())
-			} else {
-				fmt.Println("Writing MfgKey to " + filename + " for sample use ...")
-			}
-		}
+		opts.CertIdentity = CertIdentity
 	}
 
+	// Mfg Certificates
```

Comment:
Manufacturer certificates are not needed at all. As I wrote, you can drop them completely.

Reply:
@ondrejtomcik 
Manufacturer certificates is using in NewSDKClient() even if not using in Just-work OTM. I want to remain this codes to show how to use `DeviceOwnershipSDKConfig` for the future use.

Reply:
Sure, but mfg certs should be just optional.